Apps/Admin/Server/Controllers/TagController.cs

```diff
@@ -94,17 +95,17 @@ public RequestResult<AdminTagView> DeleteTag([FromBody] AdminTagView tag)
         /// <summary>
         /// Associate an existing admin tag to the feedback with matching id.
         /// </summary>
-        /// <returns>The added tag model wrapped in a request result.</returns>
+        /// <returns>Returns the associated admin tag wrapped in a request result.</returns>
         /// <param name="feedbackId">The feedback id.</param>
-        /// <param name="tag">The tag model.</param>
-        /// <response code="200">Returns the list of dependents.</response>
+        /// <param name="tagIds">A list of admin tag ids.</param>
+        /// <response code="200">The user feedback containing the added tag model wrapped in a request result.</response>
         /// <response code="401">the client must authenticate itself to get the requested response.</response>
         /// <response code="403">The client does not have access rights to the content; that is, it is unauthorized, so the server is refusing to give the requested resource. Unlike 401, the client's identity is known to the server.</response>
         [HttpPut]
         [Route("UserFeedback/{feedbackId}/[controller]")]
-        public RequestResult<UserFeedbackTagView> AssociateTag(string feedbackId, [FromBody] AdminTagView tag)
+        public RequestResult<UserFeedbackView> AssociateTag(string feedbackId, [FromBody] Collection<Guid> tagIds)
```

Comment:
Nit: we should be using `IList<>` or `IEnumerable<>` (instead of `Collection<>` or `ICollection<>`), depending on whether we require access to list and collection functionality or just need to iterate over the results.

Reply:
Changed.